WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageViewModel.swift

```diff
@@ -146,10 +151,17 @@ private extension AddProductFromImageViewModel {
         Task { @MainActor in
             do {
                 let texts = try await imageTextScanner.scanText(from: image)
+                guard texts.isNotEmpty else {
```

Comment:
This case will not be tracked at all. Should we move the tracking to before this guard?

Reply:
Good catch. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/10282/commits/f4be8ebe0ab5eafe6a6e1a714dc39686e691cc4b